app/vmagent/remotewrite/streamaggr.go

```diff
@@ -50,7 +50,7 @@ var (
 	streamAggrIgnoreOldSamples = flagutil.NewArrayBool("remoteWrite.streamAggr.ignoreOldSamples", "Whether to ignore input samples with old timestamps outside the current "+
 		"aggregation interval for the corresponding -remoteWrite.streamAggr.config at the corresponding -remoteWrite.url. "+
 		"See https://docs.victoriametrics.com/stream-aggregation/#ignoring-old-samples")
-	streamAggrIgnoreFirstIntervals = flag.Int("remoteWrite.streamAggr.ignoreFirstIntervals", 0, "Number of aggregation intervals to skip after the start "+
+	streamAggrIgnoreFirstIntervals = flagutil.NewArrayInt("remoteWrite.streamAggr.ignoreFirstIntervals", 0, "Number of aggregation intervals to skip after the start "+
```

Comment:
This requires vmagent.md to be updated as well.

Reply:
Btw, this change helped me to notice that `streamAggrDropInputLabels ` is misused.

@Haleygo @AndrewChubatiuk would you mind fixing this as well?

Reply:
do you mean this issue?
https://github.com/VictoriaMetrics/VictoriaMetrics/pull/6743

Reply:
Yes

Reply:
there're two issues in this PR, the one, which is not fixed in it, requires changes in string array flag type

Reply:
Fixed docs.